packages/widgetbook/lib/src/addons/grid_addon/grid_addon.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+
+import '../../fields/fields.dart';
+import '../common/common.dart';
+import 'grid_painter.dart';
+import 'grid_setting.dart';
+
+class GridAddon extends WidgetbookAddon<GridSetting> {
+  GridAddon([int? size])
+      : super(
+          name: 'Grid',
+          initialSetting: GridSetting(size: size ?? 5),
+        );
+
+  @override
+  List<Field> get fields => [];
+
+  @override
+  GridSetting valueFromQueryGroup(Map<String, String> group) {
+    return GridSetting(
+      size: int.tryParse(group['size'] ?? '') ?? initialSetting.size,
+    );
+  }
+
+  @override
+  Widget buildUseCase(
+    BuildContext context,
+    Widget child,
+    GridSetting setting, {
+    Key? key,
+  }) {
```

Comment:
The extra parameter `key` make `buildUseCase` an invalid override.
```suggestion
    GridSetting setting,
  ) {
```

Reply:
The extra key parameter is not in use. Was trying out some idea and must have forgot to clean up properly.